pkg/resources/testdata/destinations/azure_blob_storage.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "b8071802-63af-11ee-aee9-26dab184329f",
+  "title": "azure blob sink title",
+  "description": "azure blob sink description",
+  "account_id": "7b212506-23cb-11ed-b300-4ef12c27e273",
+  "pipeline_id": "0bf994e6-5c7e-11ee-b816-26dab184329f",
+  "generation_id": 3,
+  "type": "azure-blob-storage",
+  "deploy_type": "saas",
+  "user_config": {
+    "ack_enabled": true,
+    "batch_timeout_secs": 4000,
+    "encoding": "json",
+    "compression": "gzip",
+    "container_name": "logs",
+    "connection_string": "http://localhost:3000",
+    "prefix": "error_"
+  },
+  "inputs": [
+    "ad6ebb62-63ae-11ee-9b45-26dab184329f",
+    "79551faa-6c6a-11ee-be81-6671faf7df66"
+  ]
+}
```

Comment:
Most of these are missing a trailing newline.  You can make your editor always add one upon save...

Reply:
It's a JSON file - trailing commas are not allowed

Reply:
really?!  I don't think I've ever encountered that, but ok.

Reply:
```
➜  ~  (⎈ |a2.us-east-1.stage.logdna:default)
> echo '["a",]' | jq
parse error: Expected another array element at line 1, column 6
➜  ~  (⎈ |a2.us-east-1.stage.logdna:default)
> echo '["a"]' | jq
[
  "a"
]
```

Reply:
yeah, I think that a newline that's outside of the JSON doesn't matter

Reply:
OH!  not "trailing commas", trailing newlines--to keep github from printing the red "NO NEWLINE" warning